Sources/SwiftDiagnostics/Message.swift

```diff
@@ -30,6 +30,7 @@ public enum DiagnosticSeverity {
   case error
   case warning
   case note
+  case remark
```

Comment:
Same here, could you add this to the release notes?

Reply:
Sure